x-pack/plugins/uptime/public/components/fleet_package/http/advanced_fields.tsx

```diff
@@ -383,23 +375,13 @@ export const HTTPAdvancedFields = memo<Props>(({ validate }) => {
             />
           }
           labelAppend={<OptionalLabel />}
-          isInvalid={
-            !!validate[ConfigKeys.RESPONSE_HEADERS_CHECK]?.(
-              fields[ConfigKeys.RESPONSE_HEADERS_CHECK]
-            )
-          }
-          error={
-            !!validate[ConfigKeys.RESPONSE_HEADERS_CHECK]?.(
-              fields[ConfigKeys.RESPONSE_HEADERS_CHECK]
-            )
-              ? [
-                  <FormattedMessage
-                    id="xpack.uptime.createPackagePolicy.stepConfigure.httpAdvancedOptions.responseHeadersField.error"
-                    defaultMessage="Header key must be a valid HTTP token."
-                  />,
-                ]
-              : undefined
-          }
+          isInvalid={!!validate[ConfigKeys.RESPONSE_HEADERS_CHECK]?.(fields)}
+          error={[
```

Comment:
I do not remember why this was made an array. My gut tells me that this is not needed.